### ISSUE TITLE
Create FaceNet with Triplet Loss

### ISSUE THREAD

**Issue by jannikjw** [jannikjw/one-shot-face-recognition#19]
facenet-pytorch

Faceneot-pytorch is trained using cross-entopy softmax. 
- Why is the accuracy still so high?
- Can you finetune a model with a different loss function -> triplet loss?
- If not, how can we retrain FaceNet using triplet loss without a huge dataset.
